gate-web/src/main/groovy/com/netflix/spinnaker/gate/controllers/GremlinController.groovy

```diff
@@ -0,0 +1,32 @@
+package com.netflix.spinnaker.gate.controllers
+
+import com.netflix.spinnaker.gate.services.GremlinService
+import io.swagger.annotations.ApiOperation
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.web.bind.annotation.RequestBody
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RequestMethod
+import org.springframework.web.bind.annotation.RestController
+
+@RestController
+@RequestMapping('/gremlin')
```

Comment:
This also needs to be made conditional on gremlin being enabled, otherwise startup will fail trying to autowire the missing `GremlinService`.